recipes/mplcursors/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "mplcursors" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/m/mplcursors/mplcursors-{{ version }}.tar.gz
+  sha256: 67fbbc5c7ea979c223baf01df55e3cab425f71d3fb96ed100686873ddf3ae786
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >3.4
+    - pip
+    - setuptools_scm
+  run:
+    - python >3.4
+    - matplotlib >2.1
+
+test:
+  imports:
+    - mplcursors
+  # Cannot run the pytest suite since it isn't installed with the package.
```

Comment:
You can run it.
Just put ``pytest`` inside of the ``requires`` section. Like:

```
test:
  requires:
    - pytest
```

and also add a ``commands`` section inside of ``test`` to execute your test suite if you want to.

Reply:
Will we still be in the work directory? The test files are missing from the actual install. Upstream decided not to install them.